client/src/pages/AboutPage/AboutPage.css

```diff
@@ -0,0 +1,40 @@
+@import url("https://fonts.googleapis.com/css2?family=Aleo:wght@400;700&display=swap");
+
+body {
+  height: 100%;
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: flex-start;
+  font-family: "Aleo", serif;
+  background-color: #ffffff;
+}
+
+.about-wrapper {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  padding-top: 50px;
+  margin: 30px;
+}
+
+.about-container {
+  padding: 20px;
+  border: 4px solid #0e4749;
+  border-radius: 15px;
+  width: 1300px;
```

Comment:
1300px seems way too big for a mobile display as we are coding in a "mobile first" way.
You should use mediaqueries and relative sizes.

Reply:
Done